tests/snapshots/tests.contrib.sanic.test_sanic_server.test_multiple_requests_sanic_http_21.9_to_22.9.json

```diff
@@ -0,0 +1,76 @@
+[[
+  {
+    "name": "sanic.request",
+    "service": "sanic",
+    "resource": "GET /hello",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "web",
+    "meta": {
+      "_dd.p.dm": "-0",
+      "http.method": "GET",
+      "http.status_code": "200",
+      "http.url": "http://0.0.0.0:8000/hello",
+      "http.useragent": "python-requests/2.28.1",
+      "runtime-id": "9062d3d7bf5948e8a769aff26d44c128",
+      "sanic.route.name": "test_sanic_server.hello"
+    },
+    "metrics": {
+      "_dd.agent_psr": 1.0,
+      "_dd.top_level": 1,
+      "_dd.tracer_kr": 1.0,
+      "_sampling_priority_v1": 1,
+      "system.pid": 99552
+    },
+    "duration": 99781000,
+    "start": 1664910315709049000
+  },
+     {
+       "name": "__main__.random_sleep",
```

Comment:
instead of adding a snapshot just for sanic>=22.9.0 we could set the operation and resource name of this span: https://github.com/DataDog/dd-trace-py/blob/549ec706e9f53c3308d9a7fb7ba43cb835185eb5/tests/contrib/sanic/run_server.py#L22

Reply:
If the extra file isn't an issue I would like to keep this snapshot. It illustrates that a sanic app in v22.9.0 is now spawned in a child process.